lib/query.js

```diff
@@ -41,25 +41,25 @@ const relationIdentifierMap = {
 
 export const isRel = node => isApply(node) && node.op in relationIdentifierMap
 
-export const isNumber = node => {
```

Comment:
I guess we `isInteger`.  Okay, let's leave this as is.  We can always add it back later if it does turn out we need it.